api/src/main/java/uk/gov/dhsc/htbhf/eligibility/factory/EligibilityResponseFactory.java

```diff
@@ -49,10 +54,27 @@ public EligibilityResponse createEligibilityResponse(DWPEligibilityResponse dwpR
 
         builder.numberOfChildrenUnderOne(nullSafeMax(dwpResponse.getNumberOfChildrenUnderOne(), hmrcResponse.getNumberOfChildrenUnderOne()));
         builder.numberOfChildrenUnderFour(nullSafeMax(dwpResponse.getNumberOfChildrenUnderFour(), hmrcResponse.getNumberOfChildrenUnderFour()));
+        builder.children(createChildren(dwpResponse, hmrcResponse));
 
         return builder.build();
     }
 
+    private List<ChildDTO> createChildren(DWPEligibilityResponse dwpResponse, HMRCEligibilityResponse hmrcResponse) {
+        List<ChildDTO> dwpChildren = getListOfEmptyIfNull(dwpResponse.getChildren());
+        List<ChildDTO> hmrcChildren = getListOfEmptyIfNull(hmrcResponse.getChildren());
+
+        if (dwpChildren.size() >= hmrcChildren.size()) {
+            return dwpResponse.getChildren();
+        }
+
+        return hmrcResponse.getChildren();
+    }
+
+    // Return the list if it's not null, otherwise return any empty list.
+    private List<ChildDTO> getListOfEmptyIfNull(List<ChildDTO> children) {
```

Comment:
`getListOfEmptyIfNull` is a bit of a mouthful. How about `getEmptyListIfNull`, or `notNullList`?

Reply:
Renamed getEmptyListIfNull